api/src/pcapi/routes/public/individual_offers/v1/bookings.py

```diff
@@ -46,3 +47,25 @@ def get_booking_by_token(token: str) -> serialization.GetBookingResponse:
         raise api_errors.ApiErrors(errors={"booking": str(exc)}, status_code=403)
 
     return serialization.GetBookingResponse.build_booking(booking)
+
+
+@blueprint.v1_bookings_blueprint.route("/token/<token>", methods=["PATCH"])
+@spectree_serialize(on_success_status=204, api=blueprint.v1_bookings_schema)
+@api_key_required
+def validate_booking_by_token(token: str) -> None:
```

Comment:
J'ajouterais bien une petite docstring pour que ce soit visible sur le swagger. Un truc du genre : 

Validation of a booking.
To confirm that the booking has been successfully used by the beneficiary.

Et si tu peux en mettre sur le GET que j'ai pas modifié :
Consultation of a booking.

The countermark or token code is a character string that identifies the reservation and serves as proof of booking.

This unique code is generated for each user's booking on the application and is transmitted to them on that occasion.